lib/tasks/code_quality.rake

```diff
@@ -36,9 +36,11 @@ namespace :code_quality do
 
     desc "bundler audit"
     task :bundler_audit => :prepare do |task|
+      options = options_from_env(:audit_options)
```

Comment:
the env key should be named as `bundler_audit_options`

Reply:
please update the README to expose there is an option for `bundler_audit` task,  should describe what kind of options would be used

Reply:
@rainchen updated